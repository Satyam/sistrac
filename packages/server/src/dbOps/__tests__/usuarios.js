import {
  init,
  close
} from '../';
import {
  createUsuario,
  readUsuario,
  updateUsuario,
  deleteUsuario,
} from '../usuarios';

describe('dbOps.usuarios', () => {
  beforeAll(init);
  afterAll(close);
  it ('They should all be functions', () => {
    expect(typeof createUsuario).toBe('function');
    expect(typeof readUsuario).toBe('function');
    expect(typeof updateUsuario).toBe('function');
    expect(typeof deleteUsuario).toBe('function');
  });
});
