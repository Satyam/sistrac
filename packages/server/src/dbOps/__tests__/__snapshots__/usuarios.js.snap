// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dbOps.usuarios with mock db createUsuario fail 1`] = `"insert into Usuarios set \`usuario\` = 'pepe', \`password\` = '123456789', \`nombre\` = 'José Pérez'"`;

exports[`dbOps.usuarios with mock db createUsuario success 1`] = `"insert into Usuarios set \`usuario\` = 'pepe', \`password\` = '123456789', \`nombre\` = 'José Pérez'"`;

exports[`dbOps.usuarios with mock db deleteUsuario fail 1`] = `"delete from Usuarios where idUsuario = 42"`;

exports[`dbOps.usuarios with mock db deleteUsuario success 1`] = `"delete from Usuarios where idUsuario = 42"`;

exports[`dbOps.usuarios with mock db readUsuario fail 1`] = `"select * from Usuarios where idUsuario = \`idUsuario\` = 42"`;

exports[`dbOps.usuarios with mock db readUsuario success 1`] = `"select * from Usuarios where idUsuario = \`idUsuario\` = 42"`;

exports[`dbOps.usuarios with mock db readUsuarioPorUsuario fail 1`] = `"select * from Usuarios where usuario = \`usuario\` = 'pepe'"`;

exports[`dbOps.usuarios with mock db readUsuarioPorUsuario success 1`] = `"select * from Usuarios where usuario = \`usuario\` = 'pepe'"`;

exports[`dbOps.usuarios with mock db readUsuarios 1`] = `
"select idUsuario, usuario, nivel,
    rolGuarda, rolDios, rolSupervisor,
    rolMecanico, funcion, nombre from Usuarios "
`;

exports[`dbOps.usuarios with mock db updateUsuario fail 1`] = `"update Usuarios set \`usuario\` = 'pepe', \`password\` = '123456789', \`nombre\` = 'José Pérez' where idUsuario = 42"`;

exports[`dbOps.usuarios with mock db updateUsuario success 1`] = `"update Usuarios set \`usuario\` = 'pepe', \`password\` = '123456789', \`nombre\` = 'José Pérez' where idUsuario = 42"`;
